packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -2562,6 +2563,23 @@ describe('RpcClient test', () => {
     });
   });
 
+  describe('runView', () => {
+    it('query the right url and data', async (done) => {
+      const testData = {
+        unparsing_mode: "Readable"
+      };
+
+      await client.runView(testData as any);
```

Comment:
Can you use valid param in the test, `testData` is missing all the required properties. That way it won't need to be casted as any.

Reply:
A test also showing that we can override the default unparsing_mode can be great also.

Reply:
will amend that!